gherkin/flagd.feature

```diff
@@ -0,0 +1,81 @@
+Feature: flagd providers
```

Comment:
I've implemented most of these in a branch in the js-sdk contribs. They work as expected.